app/src/main/java/com/guerinet/mymartlet/util/dbflow/DBUtils.java

```diff
@@ -79,7 +80,7 @@
 
     /**
      * Updates the objects in a DB by updating existing objects, removing old objects, and
-     *  inserting new ones.
+     * inserting new ones.
```

Comment:
Continuation of formatting preferences: I add a space to sentences that take up multiple lines in a comment. Hitting the tab button here also does the trick, as it brings you just one column off.